lib/pbench/test/functional/agent/cli/commands/tools/test_tools_list.py

```diff
@@ -54,15 +116,15 @@ def test_no_args(self, tool, agent_config):
         out, err, exitcode = pytest.helpers.capture(command)
         assert TRACEBACK not in err
         assert EMPTY == err
-        assert b"default: testhost.example.com ['perf']" in out
+        assert b"group: default; host: testhost.example.com; tools: perf\n" in out
         assert exitcode == 0
 
     def test_name(self, tool, agent_config):
         command = ["pbench-list-tools", "-n", "perf"]
         out, err, exitcode = pytest.helpers.capture(command)
         assert TRACEBACK not in err
         assert EMPTY == err
-        assert b"tool name: perf groups: default" in out
+        assert b"group: default; host: testhost.example.com; tools: perf" in out
```

Comment:
What about `mpstat` in this one?  (It's supposed to be _absent_, right?  Shouldn't we assert that?)

Reply:
Ditto line 136, 156, et al.

Reply:
Yes, I changed all of them to require equality, rather than inclusion.